modules/apps/jaggery/mdm/src/mdm/units/login/public/js/login-box.js

```diff
@@ -1,3 +1,37 @@
+
+function utf8_to_b64( str ) {
+    return window.btoa(unescape(encodeURIComponent( str )));
```

Comment:
this is a know function for b64 conversion, but both unescape and escape are  deprecated. try to use latest